src/sagemaker/serve/model_server/multi_model_server/server.py

```diff
@@ -20,6 +20,82 @@
 logger = logging.getLogger(__name__)
 
 
+class InProcessMultiModelServer:
+    """In Process Mode Multi Model server instance"""
+
+    def _start_serving(
+        self,
+        client: object,
+        image: str,
+        model_path: str,
+        secret_key: str,
+        env_vars: dict,
+    ):
+        """Initializes the start of the server"""
+        env = {
+            "SAGEMAKER_SUBMIT_DIRECTORY": "/opt/ml/model/code",
+            "SAGEMAKER_PROGRAM": "inference.py",
+            "SAGEMAKER_SERVE_SECRET_KEY": secret_key,
+            "LOCAL_PYTHON": platform.python_version(),
+        }
+        if env_vars:
+            env_vars.update(env)
+        else:
+            env_vars = env
+
+        self.container = client.containers.run(
```

Comment:
Are we spinning up a docker container or using fast api for serving?

Reply:
The container will be stubbed, this is only 1/2 of the full implementation of InProcess mode. My next PR will include the FastAPI.